cmd/expiration-mailer/main.go

```diff
@@ -168,13 +174,13 @@ func (m *mailer) findExpiringCertificates() error {
 			 AND cert.expires > :cutoffA
 			 AND cert.expires <= :cutoffB
 			 AND cs.status != "revoked"
-			 AND cs.lastExpirationNagSent <= :nagCutoff
+			 AND COALESCE(TIMESTAMPDIFF(SECOND, cs.lastExpirationNagSent, cert.expires) > :nagCutoff, 1)
```

Comment:
Yes, if lastExpirationNagSent is null then diff returns null. I think lastExp should never be null, but if it is, surely it means "never notified", not "please skip all notifications."